article/app/model/KeyEventData.scala

```diff
@@ -24,10 +24,18 @@ object KeyEventData {
     val TimelineMaxEntries = 7
     val timelineBlocks = blocks.sortBy(_.publishedCreatedTimestamp).reverse.take(TimelineMaxEntries)
     timelineBlocks.map { bodyBlock =>
-      KeyEventData(bodyBlock.id, bodyBlock.referenceDateForDisplay().map(LiveBlogDate(_, timezone)), bodyBlock.title)
+      /*
+    Composer should set a default title of `Summary` on summary blocks but instead it allows title to be empty (None).
+    Until this work is complete, the following pattern match ensures we set a default title of `Summary`
+       */
+      val title = bodyBlock.eventType match {
+        case SummaryEvent => bodyBlock.title.getOrElse("Summary")
+        case _            => bodyBlock.title.getOrElse("")
```

Comment:
my only concern here is that we may end up with an empty h2 tag in the html? Perhaps we should consider keeping the title optional and just populating it with a Some("Summary") when the event type is summary

Reply:
Good point, I'll revise this!